test/webgl/texture.spec.js

```diff
@@ -214,3 +214,66 @@ test('WebGL2#Texture2D setParameters', t => {
 
   t.end();
 });
+
+test('WebGL#Texture2D NPOT Workaround: texture creation', t => {
+  const {gl} = fixture;
```

Comment:
Wrote test cases for WebGL2, there are no exceptions to test, but I am testing parameter settings (for WebGL2, we shouldn't change these settings).